wdns/str_to_rdata_ubuf.c

```diff
@@ -1069,7 +1069,7 @@ _wdns_str_to_rdata_ubuf(ubuf *u, const char *str,
 				my_rrtype = wdns_str_to_rrtype(s_rrtype);
 				free(s_rrtype);
 
-				if (my_rrtype == 0 || (rrtype >= 128 && rrtype < 256) || rrtype == 65535) {
```

Comment:
The removal of this rrtype constraint should be noted in the commit message, as it's a change in approach.

The constraints would be appropriate for a DNS server where including these reserved / query-only types would be an application-level error, but _our_ priority is faithfully displaying bitmaps we see in the wild.

(Normally, I'd say such changes in approach should be reflected in comments, but in this case that would be a comment on removed code, which can be confusing.)